ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml

```diff
@@ -0,0 +1,69 @@
+---
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
+  tasks:
+    - name: Deploy content provider registry
+      ansible.builtin.include_role:
+        name: registry_deploy
+
+    - name: Call repo setup
+      ansible.builtin.import_role:
+        name: repo_setup
+      vars:
+       cifmw_repo_setup_output: "/etc/yum.repos.d/"
+
+    - name: Get latest commit when no PR is provided
+      ansible.builtin.command:
+        cmd: git show-ref --head --hash head
+      args:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/edpm-image-builder"
+      register: git_head_out
+
+    - name: Set pr_sha to be used as image tag
+      ansible.builtin.set_fact:
+        pr_sha: "{{ pr_commit_hash_sha | default(git_head_out.stdout | trim) }}"
+        cacheable: true
+
+    - name: Build edpm and ipa images
+      ansible.builtin.include_role:
+        name: edpm_build_images
+      vars:
+        cifmw_edpm_build_images_tag: "{{ pr_sha }}"
+
+    - name: Push edpm-hardened-uefi image to registry
+      containers.podman.podman_image:
+        name: "edpm-hardened-uefi"
+        push_args:
+          dest: "{{ cifmw_rp_registry_ip | default('localhost') }}:5001"
+        tag: "{{ pr_sha }}"
+        push: yes
+
+    - name: Push ironic-python-agent image to registry
+      containers.podman.podman_image:
+        name: "ironic-python-agent"
+        push_args:
+          dest: "{{ cifmw_rp_registry_ip | default('localhost') }}:5001"
+        tag: "{{ pr_sha }}"
+        push: yes
+
+    - name: Set build images output
```

Comment:
You can try something like below

~~~
    - name: Set cifmw_build_images_output
      set_fact:
        cifmw_build_images_output:
          edpm-hardened-uefi:
            image: "{{ cifmw_rp_registry_ip | default('localhost') }}/edpm-hardened-uefi:{{ pr_sha }}"
          ironic-python-agent:
            image: "{{ cifmw_rp_registry_ip | default('localhost') }}/ironic-python-agent:{{ pr_sha }}"
~~~